ReadMe.md

```diff
@@ -47,6 +47,9 @@ GeoBlazor brings the power of the ArcGIS Maps SDK for JavaScript into your Blazo
 
 [![Discord](https://img.shields.io/discord/1027907220949717033?color=%235865F2&label=chat&logo=discord&logoColor=white)](https://discord.gg/hcmbPzn4VW)
 
+## ‼️Important Version Information
+The below documentation is for GeoBlazor version 4.x. To view the documentation for previous versions, switch to the main branch, or go to the [Documentation](https://docs.geoblazor.com) page.
+
```

Comment:
This will be out of date as soon as we release v4, because we will merge into main. I suggest we point them to https://www.nuget.org/packages/dymaptic.GeoBlazor.Core/3.2.1#readme-body-tab, which will remain permanently locked to v3.